### PR TITLE
Use pyinstaller for packaging, replacing the cx_Freeze + NSIS combo.

### PR DESCRIPTION
**Pull request by jamsilva**
With this change I'm able to produce working self-contained executables for Windows (fixing the issues with the 3.33 Windows release) and also for Linux. I tested decompilation, script extraction and recompilation with both and didn't notice any problems.

### CODE REVIEW

**Comment by vince94** [comment]
Whoa, thanks for contributing!! If you'd like to join a slack chat where a bunch of EB hackers hang out, let me know and I'll get an invite link for you 👍

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -6,47 +6,69 @@ Because Python is notorious for being difficult to maintain a clean installation
...
 ## Windows
 
 1. Install:
-    1. [Python 3.6](https://www.python.org/downloads/release/python-364/) (64-bit version)
-    1. [Visual C++ 2017 Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017)
-    1. [PyYAML](http://pyyaml.org/wiki/PyYAML)
-        1. `pip install PyYAML-xxx-win_amd64.whl`
+    1. [Python 3.7](https://www.python.org/downloads/release/python-374/) (32-bit version)
+    1. [Visual C++ 2019 Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16)
+    1. Select "C++ build tools" under the "Workloads" tab and make sure these are ticked:
```
Comment (mrtenda) [inline DEVELOPMENT.md]: Where is this "Workloads" tab? Is it part of the Python installer, or the Visual C++ 2019 Build Tools installer?

File: setup.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
     ],
     dependency_links=[
         "https://github.com/Lyrositor/CCScriptWriter/tarball/master#egg=CCScriptWriter-1.2",
-        "https://github.com/tripped/ccscript_legacy/tarball/master#egg=ccscript-1.338"
+        "https://github.com/jamsilva/ccscript_legacy/tarball/master#egg=ccscript-1.338"
```
Comment (mrtenda) [inline setup.py]: Have you seen this PR? Perhaps you might consider merging it into your branch of ccscript_legacy? It has been a thorn in the sides of many fan translators who use Windows with non-English languages.

https://github.com/tripped/ccscript_legacy/pull/7/files

**Comment by jamsilva** [comment]
Thanks for the comments. I'll look into that PR and clarify the DEVELOPMENT.md file - I forgot to indent that line to show that it is part of the VC++ tools...

File: setup.py (hunk 2 above)
Comment (jamsilva) [reply setup.py]: Merged.

File: DEVELOPMENT.md (hunk 1 above)
Comment (jamsilva) [reply DEVELOPMENT.md]: Clarified the instructions and reorganized the file to combine the things that are common to both Windows and Linux.

**Pull request merged**